internal/pkg/fixtures/configs/im-transformation.json

```diff
@@ -0,0 +1,24 @@
+{
+  "componentId": "keboola.snowflake-transformation",
+  "name": "im-transformation",
+  "configuration": {},
+  "rows": [
+    {
+      "name": "test",
+      "isDisabled": false,
+      "configuration": {
+        "parameters": {},
+        "storage": {
+          "input": {
+            "tables": [
+              {
+                "source": "%%TEST_IM_TRANSFORMATION%%.table",
+                "destination": "table"
+              }
```

Comment:
V tomto commite pridavam dalsi test.
Mame tu 2 configy, tento s IM, a druhy s OM.

V teste simulujem, ze config ma OM do `in.c-my-super-bucket-<ID>.table`.
Tj. nejaka custom bucket a autor sablony si dal do nazvu ID konfiguracie.
Transformacia ma tu istu bucket v IM.

To sa moze pouzit pri komponentach co nemaju default bucket.
No ked sa sablona aplikuje viac-krat, tak kazda "instancia" musi mat unikatnu bucket.